### DISCUSSION TITLE
2.3.17

### DISCUSSION THREAD

**Discussion by benhillis** [microsoft/WSL#11911] [General]
*  Revert Linux kernel to 5.15.153.1-2 while issues are being debugged
*  Fix race condition when WSL2 distribution is stopped
*  Use a format attribute to catch printf format errors compile time
*  Fix segfault in init when boot.initPath doesn't exist (solves #11725)
*  Add default Hyper-V firewall rules to allow inbound mDNS traffic (solves #11852)
*  Update Windows SDK to version 26100
*  Update Microsoft.Windows.ImplementationLibrary to version 1.0.240803.1

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/WSL/releases/tag/2.3.17'>2.3.17</a>.</em>

**Comment by SledgeHammer01** [comment]
Is it expected that PV/PVC mounts error out with "Bad Message"? I tried installing Airflow via Helm and its unable to mount PVCs.

**Comment by vikash236** [comment]
Feature Suggestion for WSL
Hi there,

I’ve been using WSL and really appreciate how it bridges the gap between Windows and Linux. I have a few ideas that I think could make WSL even better, especially for people working in fields like data science and AI:

1. Central Management Hub: It would be amazing to have a central application within WSL that manages all other applications, memory, storage, and performance, similar to a desktop environment on Windows but optimized for WSL. This would make it much easier to keep everything running smoothly.
This tool could make it easier to:

- Manage Applications: Launch and control Linux apps in one place.

- Optimize Memory: Keep track of and free up memory as needed.

- Monitor Storage: See how much space is being used and clean up easily.

- Improve Performance: Automatically adjust settings for better performance.


2. Linking C Compilers and Toolkits: It would be great if WSL could link C compilers (like gcc and g++), the NVIDIA Toolkit, and integrate Conda and Pip environments into one unified folder. This would streamline the workflow for developers and researchers by integrating the best of both Windows and Ubuntu environments.

3.Integrated Components: Introducing more integrated components between WSL and Windows could greatly improve performance and efficiency. For example, seamless integration between the Linux and Windows file systems, or tools that leverage both environments simultaneously, could be a game-changer for productivity.

I believe these features could really help a lot of people, especially those working in data science, AI, and other tech fields. They would make WSL more powerful and user-friendly.

Thanks for considering these ideas!

**Comment by ankurdotcom** [comment]
System Detail :

OS Name	Microsoft Windows 10 Pro

Version	10.0.19045 Build 19045
System Type	x64-based PC


------------------------------------------------------------

I get this error while installing WSL on my Windows machine.

-------------------------------------------------------------
  [Window Title]
Network Error

[Main Instruction]
Windows cannot access \\wsl.localhost

[Content]
Check the spelling of the name. Otherwise, there might be a problem with your network. To try to identify and resolve network problems, click Diagnose.

[V] See details  [Diagnose] [Cancel]

[Expanded Information]
Error code: 0x800704b3
The network path was either typed incorrectly, does not exist, or the network provider is not currently available. Please try retyping the path or contact your network administrator.
